src/op_mode/show_nat66_rules.py

```diff
@@ -60,12 +68,10 @@
         else:
             if 'masquerade' in data['expr'][3]:
                 tran_addr = 'masquerade'
-            elif 'log' in data['expr'][3]:
-                continue
             else:
                 tran_addr = dict_search('snat.addr' if args.source else 'dnat.addr', data['expr'][3])
         
-        print(format_nat66_rule % (comment, srcdest, tran_addr, interface))
+        print(format_nat66_rule % (rule, srcdest, tran_addr, interface))
```

Comment:
Can we mobe to Python3 format identifiers?

Reply:
done